example.json

```diff
@@ -0,0 +1,21 @@
+{
+    "_id": "c-35",
+    "_parentId":"b-20",
+    "_type":"component",
+    "_classes": "",
+    "_layout": "full",
+    "_component": "vimeo",
+    "title": "Component Title",
+    "displayTitle": "Component Title",
+    "body": "This is a vimeo component",
+    "instruction":"Click the button to play",
+    "_setCompletionOn": "play",
+    "_media": {
+        "_source": "https://vimeo.com/319977833",
```

Comment:
might be better to use a protocol-less url here i.e. `"_source": "//vimeo.com/319977833"`